src/App.js

```diff
@@ -1,9 +1,34 @@
-function App () {
+import birdData from "./data/birds";
+import bonusItems from "./data/bonusItems";
+import "./App.css";
+import { useState } from "react";
+import Birdcards from "./components/Birdcards";
+import Cart from "./components/Cart";
+import Checkout from "./components/Checkout";
+
+function App() {
+
+  const [birds, setBirds] = useState([]);
+
+  function handleBirdCount(bird){
+    const birdObj = {...bird};
+    setBirds([...birds, birdObj])
+  }
```

Comment:
At first, I didn't understand why you copy the bird into a new object, but now I see that this allows you to create a new object for them which allows the deleting implementation you chose to work! Let me know if what I wrote here makes sense cause if not I would like to explain it better!